app/src/main/java/com/alphawallet/app/widget/ActionSheetDialog.java

```diff
@@ -672,14 +673,26 @@ public void gotAuthorisation(boolean gotAuth)
                 {
                     confirmationWidget.startProgressCycle(4);
                 }
-                if (mode == ActionSheetMode.SEND_TRANSACTION)
-                {
-                    actionSheetCallback.sendTransaction(formTransaction());
-                }
-                else
+
+                switch (mode)
```

Comment:
@seabornlee can we remove the 'default' check for when the switch is on an enum? Because; this way if we add a new element to the enum we can get a build error.

Reply:
To achieve that, you can add `@SuppressWarnings("checkstyle:MissingSwitchDefault")` on top of `switch`.

Reply:
> To achieve that, you can add `@SuppressWarnings("checkstyle:MissingSwitchDefault")` on top of `switch`.

I didn't realise the lint check took into account those directives. That's really handy.